### ISSUE TITLE
Login improvements

### ISSUE THREAD

**Issue by aboydnw** [NASA-IMPACT/nasa-apt#469]
Miro board detailing current workflow:
https://miro.com/app/board/uXjVOIZJqw8=/?invite_link_id=923746756679

AC:
- [x] Once user signs in through cognito, drop them on the APT dashboard signed into their account
- [x] Subheader on Dashboard of unapproved user is updated to say "Your account is not yet approved by an APT Administrator, this action can take up to 1 business day. If you have not been approved after this time, please send a message via the feedback button with your name, email and the day/time you signed up."
- [x] The verification email is updated to include a link to APT (see text below).

Verification email
> Thank you for signing up to the Algorithm Publication Tool.
Please verify you account by clicking on [Verify Email](https://nasa-apt-api-lambda-staging.auth.us-east-1.amazoncognito.com/confirmUser?client_id=5lbh4h6aooego0rkffi36hakhg&user_name=441adbbf-d761-4bbb-a7e9-a8fd7be63b06&confirmation_code=981684).
Your account only needs to be verified once. If your account is already verified, you can return to the [Algorithm Publication Tool](https://nasa-apt.surge.sh) to sign in. 
Sincerely,
The NASA APT team

**Comment by aboydnw** [comment]
@deborahUAH see the Miro link above for the login flow. Let me know what you think of the changes I outlined in the acceptance criteria (AC). In particular, I am curious about your thoughts on #4. If there are any more prescriptive instructions we can provide people on how to get their accounts approved, that would be ideal.

**Comment by deborahUAH** [comment]
@aboydnw I looked at Miro board.  I agree with your suggestions.  Anything that will reduce the repetitive tasks when  logging in after signing up.  Regarding AC#4,  I would prefer giving them a reasonable time estimate... something like "Your account is not yet approved by an APT Administrator, this action can take up to 1 business day.  If you have not been approved after this time, please send a message via the feedback button <link if possible> with your name, email and the day/time you signed up."

**Comment by danielfdsilva** [comment]
@aboydnw There seems to be little that can be done to the cognito UI. They provide the hosted UI as is to speed things up. You can customize it with some CSS to make it prettier, but functionality wise, if you want something custom, you'll have to implement it.

It is not possible to add redirects or change the flow that the hosted UI goes through.
The currently intended behavior would be:
- User signs in and then closes that tab (interaction with cognito stops here)
- User goes to the email and clicks the confirmation link
- User goes to the apt site and logs in. (the email message can be improved stating that the user has to go back to APT)

A possible improvement would be to login the user in APT from cognito (like your suggestion). Cognito is sending back an access token. Not sure why it is not being picked up. Should be relatively straightforward to implement.

Alternatively, the whole login flow can be moved to ATP. This would make the experience cohesive but as there are several flows (signup, activation, password recovery) to migrate, it is a more time consuming task.

**Comment by aboydnw** [comment]
@danielfdsilva @deborahUAH thanks for reviewing this ticket. based on what is possible in cognito and the limited time we have, I updated the AC to match. I also added an AC to update the email content. Since we can't adjust the error message in cognito when a user clicks the verification link twice (or clicks it after they have already been manually confirmed) I think we should include a link to APT so they have some direction on what to do next. This is probably just generally helpful to include as well. 

In the ticket I linked to the staging site, but this should of course point to the production site when in production.
